packages/flutter/lib/src/material/navigation_drawer.dart

```diff
@@ -148,9 +148,10 @@ class NavigationDrawer extends StatelessWidget {
       shadowColor: shadowColor,
       surfaceTintColor: surfaceTintColor,
       elevation: elevation,
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: wrappedChildren,
+      child: SafeArea(
```

Comment:
So for SafeArea & scrolling widgets, I think we typically pad the top and sides. The bottom usually allows scrolling content to render through to un-safe areas, since it can scroll up into the interactive area. I am not sure if there is a special case though for this particular widget where you would want to have the bottom safe area cut off.

The directional SafeArea parameters let you specify which sides should or shouldn't pad.